### DISCUSSION TITLE
Plotting color charts in specific color space?

### DISCUSSION THREAD

**Discussion by ssh4net** [colour-science/colour#885] [Q&A] [answered]
Sorry, I'm new in colour-science package, but can't guess where in manual info about this.

In all tutorials everything plotting in sRGB color space.
But what if I want plot color chart in ACEScg or ProPhotoRGB?

What the way to do this?

**Comment by thomasmansencal** [comment, marked as answer]
Hi @ssh4net,

All the conversions for display should go through the colourspace defined in the `colour.plotting.CONSTANTS_COLOUR_STYLE` attribute here:

https://github.com/colour-science/colour/blob/develop/colour/plotting/common.py#L112

If you change it for another RGB colourspace, the conversions will be done for that RGB colourspace. It is not very well tested, so please let us know if any issues!

Cheers,

Thomas

**Reply by ssh4net** [reply]
Thank you!
Sound that this work!

Will be great to add some examples into guides. 

Using defined aliases

```
from colour.plotting import *
from colour.models import RGB_COLOURSPACES

CONSTANTS_COLOUR_STYLE.colour.colourspace = RGB_COLOURSPACES['aces']

plot_single_colour_checker(colour_checker='ColorChecker24 - After November 2014', text_kwargs={'visible': False})
```

And using names

```
from colour.plotting import *
from colour.models import RGB_COLOURSPACES
import colour.models.rgb.datasets

CONSTANTS_COLOUR_STYLE.colour.colourspace = RGB_COLOURSPACES[colour.models.rgb.datasets.RGB_COLOURSPACE_ACES2065_1.name]

plot_single_colour_checker(colour_checker='ColorChecker24 - After November 2014', text_kwargs={'visible': False})
```

**Reply by thomasmansencal** [reply]
Thanks for reporting back @ssh4net! This certainly could be put in our [How-To Guide](https://colab.research.google.com/drive/1NRcdXSCshivkwoU2nieCvC3y14fx1X4X#sandboxMode=true)!
